addons/hr_expense/tests/test_expenses.py

```diff
@@ -538,26 +538,216 @@ def test_expense_corner_case_changing_employee(self):
         expense_employee_2.employee_id = self.expense_employee.id
         self.assertEqual(expense_employee_2.sheet_id.id, False, 'Sheet should be unlinked from the expense')
 
-    def test_expenses_corner_case_with_tax_and_lock_date(self):
-        """ Test that when creating an expense move in a locked period still works but its accounting date is the current day """
-        self.env.company.tax_lock_date = '2022-01-01'
+    def test_computation_expense_report_date_based_most_recent_expense_today(self):
+        # Test with most recent expense is today
```

Comment:
- The explanation is a bit unclear to me, could you improve it?

- I may be wrong but I thought the test explanation were supposed to be in docstring but I didn't find any documentation in PEP8 to back that.